javascript/chronometer.js

```diff
@@ -1,26 +1,40 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    setInterval(
+      () => this.currentTime++, 1000)
   }
+
+
   getMinutes() {
-    // ... your code goes here
+    // current time moet naar minuten
+    let minutes = this.currentTime / 60
+    return Math.floor(minutes)
   }
+
   getSeconds() {
-    // ... your code goes here
+    // minutes moeten naar seconden, dus *60
+    let seconds = this.currentTime - this.getMinutes() * 60;
+    return seconds;
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+  twoDigitsNumber(number) {
+    if (number < 10) {
+      return `0${number}`
+    } else { return number }
   }
+
```

Comment:
How to write the same solution using the ternary operator we introduced yesterday
```
return number < 10 ? `0${number}`: number;